Translate "peer_ledger_snapshot.md" to Japanese

takayuki-nagai opened the pull request:
Translate "peer_ledger_snapshot.md" to Japanese. This patch is related to the issue https://github.com/hyperledger/fabric-docs-i18n/issues/911.

takayuki-nagai committed:
Translate "peer_ledger_snapshot.md" to Japanese

Signed-off-by: takayuki-nagai <takayuki.nagai.nu@hitachi.com>

yukiknd commented on docs/locale/ja_JP/source/peer_ledger_snapshot.md:
```diff
@@ -1,171 +1,171 @@
+2. 個々のブロックを取得して処理することなく、特定のブロック番号時点での最小限の台帳データを含む「スナップショット」からチャネルに参加する。
 
-While the first method represents a more comprehensive way of joining a channel, because of the size of established channels (which can reach many thousands of blocks), it can take a long time for peers to pull and process all the blocks already committed to the channel. Peers that join a channel this way must also store every block since the creation of the channel, increasing storage costs for  ...
+最初の方法は、チャネルに参加する一般的な方法ですが、構築済のチャネルのサイズ(数千ブロックに達する可能性あり)によっては、ピアがチャネルにコミットされたすべてのブロックをプルして処理するのに時間がかかる場合があります。この方法でチャネルに参加するピアは、チャネル作成以降のすべてのブロックを保存する必要があり、組織のストレージコストが増加します。さらに、スナップショットによる参加は、ピアに最新のチャネル構成を提供します。これは、ジェネシスブロック以降にチャネル構成が変更された場合、重要な情報となる可能性があります。例えば、ピアはオーダーリングサービスからブロックを正常に取得する前に、最新のチャネル構成からorderer endpointやCA証明書を取得する必要がある場合があります。
```
```suggestion
最初の方法は、チャネルに参加する一般的な方法ですが、構築済のチャネルのサイズ(数千ブロックに達する可能性あり)によっては、ピアがチャネルにコミットされたすべてのブロックをプルして処理するのに時間がかかる場合があります。この方法でチャネルに参加するピアは、チャネル作成以降のすべてのブロックを保存する必要があり、組織のストレージコストが増加します。さらに、スナップショットによる参加は、ピアに最新のチャネル設定を提供します。これは、ジェネシスブロック以降にチャネル設定が変更された場合、重要な情報となる可能性があります。例えば、ピアはオーダーリングサービスからブロックを正常に取得する前に、最新のチャネル設定からorderer endpointやCA証明書を取得する必要がある場合があります。
```

yukiknd commented on docs/locale/ja_JP/source/peer_ledger_snapshot.md:
```diff
@@ -1,171 +1,171 @@
-* If your endorsement requests or queries don't require the latest block commits, you can target a peer that generates snapshots.
-* If your endorsement requests or queries require the latest block commits, you can utilize service discovery to identify and target peers with the highest block heights on a channel, thereby avoiding any peer that is generating a snapshot. Alternatively, you could utilize dedicated peers for snapshot, and not make these peers available for endorsements and queries, for example by not setting `pe ...
-* You may not want to make a peer available for endorsements and queries until it has joined all the expected channels, and has reconciled all the private data that it is authorized to receive.
+* ジェネシスブロック、スナップショットのどちらを使ってピアに参加するかを決定する際は、ジェネシスブロックからのブロック数に基づいてピアがチャネルに参加するまでに要する時間、ピアがジェネシスブロックの元のチャネル構成に基づいてオーダーリングサービスからブロックを取得できるかどうか、およびチャネルの全履歴(過去のブロック、トランザクション、ステート情報)をクエリする必要があるかどうかを考慮してください。
```
```suggestion
* ジェネシスブロック、スナップショットのどちらを使ってピアに参加するかを決定する際は、ジェネシスブロックからのブロック数に基づいてピアがチャネルに参加するまでに要する時間、ピアがジェネシスブロックの元のチャネル設定に基づいてオーダーリングサービスからブロックを取得できるかどうか、およびチャネルの全履歴(過去のブロック、トランザクション、ステート情報)をクエリする必要があるかどうかを考慮してください。
```

yukiknd commented on docs/locale/ja_JP/source/peer_ledger_snapshot.md:
```diff
@@ -1,171 +1,171 @@
-1. **Schedule a snapshot**. These snapshots must be taken at **exactly the same ledger height** on each peer. This will allow an organization to evaluate the snapshots to make sure they contain the same data. This ledger height must be equal or higher than the current block height (snapshots scheduled for a higher block height will be taken when the block height is reached). They cannot be taken  ...
-2. **When the ledger height is reached, the snapshot is taken by the peer**. The snapshot is comprised of a directory that includes files that contain the public state, hashes of private state, transaction IDs, and the collection config history. A file containing metadata relating to these files is also included. For more information, check out [contents of a snapshot](#contents-of-a-snapshot).
-3. **If the snapshot will be used by a new organization, the snapshot is sent to them**. This must be completed out of band. Because snapshot files are not compressed, it is likely that peer administrators will want to compress these files before sending them. In a typical scenario, the administrator will receive the snapshot from one of the existing organizations but will want to receive the sna ...
+1. **スナップショットをスケジュールする**。それぞれのスナップショットは、各ピアで**正確に同じ台帳の高さ**で取得する必要があります。これにより、組織はスナップショットを比較して同じデータが含まれていることを確認できます。台帳の高さは、現在のブロック高と同じかそれ以上でなければなりません(より高いブロック高でスケジュールされたスナップショットは、指定したブロック高に到達した時点で取得されます)。より低いブロック高からスナップショットを取得することはできません。現在の高さより低い高さでスナップショットをスケジュールしようとすると、エラーが発生します。チャネルに参加するためにスナップショットを使用したピアも、スナップショットを取得するために使用できます。スナップショットは必要に応じてスケジュールできるほか、組織間で合意して定期的な間隔でスケジュールすることもできます(例えば10,00 ...
```
```suggestion
1. **スナップショットをスケジュールする**。それぞれのスナップショットは、各ピアで**正確に同じ台帳の高さ**で取得する必要があります。これにより、組織はスナップショットを比較して同じデータが含まれていることを確認できます。台帳の高さは、現在のブロック高と同じかそれ以上でなければなりません(より高いブロック高でスケジュールされたスナップショットは、指定したブロック高に到達した時点で取得されます)。より低いブロック高からスナップショットを取得することはできません。現在の高さより低い高さでスナップショットをスケジュールしようとすると、エラーが発生します。チャネルに参加するためにスナップショットを使用したピアも、スナップショットを取得するために使用できます。スナップショットは必要に応じてスケジュールできるほか、組織間で合意して定期的な間隔でスケジュールすることもできます(例えば10,000ブロックごと)。これにより、一貫性があり最新のスナップショットが常に利用可能になります。ただし、再帰的なスナップショットをスケジュールすることはできません。各スナップショットは独立してスケジュールする必要があります。ただし、スケジュールできるスナップショットの数に制限はありません。スナップショットからピアに参加する際は、最新のチャネル構成ブロックの高さよりも新しいスナップショットを使用することが推奨されます。 これにより、ピアはorderering service endpointやCA証明書を含む最新のチャネル設定を取得できます。
```

yukiknd commented on docs/locale/ja_JP/source/peer_ledger_snapshot.md:
```diff
@@ -1,171 +1,171 @@
+  * スナップショットの最後のブロックまでの間にチャネルで使用されたトランザクションIDのリストからなります。このトランザクションIDは、別のトランザクションでトランザクションIDが再利用されていないことをピアが確認できるように含まれています。
 * **Collection config history**
-  * This contains the history of the collection configurations for all chaincodes. Recall from our documentation on private data that the collection configuration derives the endorsement and dissemination policies for private data collections.
+  * すべてのチェーンコードのコレクション構成の履歴を含んでいます。プライベートデータに関するドキュメントで説明したように、コレクション構成はプライベートデータコレクションの承認および配信ポリシーを導出します。
```
```suggestion
  * すべてのチェーンコードのコレクション設定の履歴を含んでいます。プライベートデータに関するドキュメントで説明したように、コレクション設定はプライベートデータコレクションの承認および配信ポリシーを導出します。
```

yukiknd reviewed (commented):
@takayuki-nagai 翻訳ありがとうございます。固有用語について、翻訳用語集に従い、訳語の統一をお願いします。
channel configuration -> チャネル設定
collection configuration -> コレクション設定

takayuki-nagai committed:
Update docs/locale/ja_JP/source/peer_ledger_snapshot.md

Co-authored-by: Yuki Kondo <57741286+yukiknd@users.noreply.github.com>
Signed-off-by: takayuki-nagai <57936858+takayuki-nagai@users.noreply.github.com>

Update docs/locale/ja_JP/source/peer_ledger_snapshot.md

Co-authored-by: Yuki Kondo <57741286+yukiknd@users.noreply.github.com>
Signed-off-by: takayuki-nagai <57936858+takayuki-nagai@users.noreply.github.com>

Update docs/locale/ja_JP/source/peer_ledger_snapshot.md

Co-authored-by: Yuki Kondo <57741286+yukiknd@users.noreply.github.com>
Signed-off-by: takayuki-nagai <57936858+takayuki-nagai@users.noreply.github.com>

Update docs/locale/ja_JP/source/peer_ledger_snapshot.md

Co-authored-by: Yuki Kondo <57741286+yukiknd@users.noreply.github.com>
Signed-off-by: takayuki-nagai <57936858+takayuki-nagai@users.noreply.github.com>

訳語の統一(collection configuration -> コレクション設定)

Signed-off-by: takayuki-nagai <takayuki.nagai.nu@hitachi.com>

takayuki-nagai commented:
@yukiknd コメントありがとうございます。ご指摘いただいた部分以外にも修正必要な個所があり(34行目と140行目)、追加で修正いたしました。また、一連のコミットをsquashしました。

yukiknd reviewed (approved):
@takayuki-nagai 翻訳と修正ありがとうございます。LGTM!
